source/views/guides/getting-started.html.slim

```diff
@@ -61,13 +67,16 @@ main
           project's code directly, we don't add it to `Cargo.toml`. Instead, we
           just install it on our system.
 
+          The CLI has the same features as the diesel library and you should pass
+          the same as you passed in `Cargo.toml`.
+
           [diesel-cli]: https://github.com/sgrif/diesel/tree/master/diesel_cli
 
         .demo__example
           .demo__example-browser
             pre.demo__example-snippet
               code
-                | cargo install diesel_cli
+                | cargo install diesel_cli --no-default-features --features postgres
```

Comment:
I'd like to keep the default because this is about a global tool, not a project-specific install. `--no-default-features` should not be a general suggestion. We can add a paragraph below with the syntax to select support only for specific backends in `cargo instal`.